package.json

```diff
@@ -34,9 +34,11 @@
     "bluebird": "^3.4.0",
     "m3u8-parser": "^4.3.0",
     "mkdirp": "^0.5.1",
+    "mpd-parser": "^0.8.1",
     "pessimist": "^0.3.5",
     "request": "^2.87.0",
-    "requestretry": "^2.0.0"
+    "requestretry": "^2.0.0",
+    "xmldom": "^0.1.27"
```

Comment:
`mpd-parser` uses `DOMParser` in the browser, but we don't have that on nodejs. luckily `xmldom` provides it.